app/helpers/miq_ae_class_helper.rb

```diff
@@ -41,7 +44,7 @@ def class_prefix(cls)
     case cls.to_s.split("::").last
     when "MiqAeClass"
       "aec"
-    when "MiqAeNamespace"
+    when "MiqAeDomain", "MiqAeNamespace"
```

Comment:
ok, @h-kataria, thanks. I'm just trying to understand how we remove these hard coded class names as each new class will need entries here and I don't really understand why.